### DISCUSSION TITLE
Buttons and cuatom header

### DISCUSSION THREAD

**Discussion by AlexCernik** [react-navigation/react-navigation#12269] [Q&A]
Hi, the button or pressable or custom header not work correctly in header, any idea? I use latest version of react native, react navigation and react native screens Thanks

**Comment by satya164** [comment]
It's a bug in react-native-screens when using new architecture https://github.com/software-mansion/react-native-screens/issues/2219

To wokraround, you can temporarily disable the new architecture.
